tools/x-repo-types/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.20
 replace github.com/algorand/go-algorand => ../..
 
 require (
-	github.com/algorand/go-algorand v0.0.0-20230502140608-e24a35add0bb
+	github.com/algorand/go-algorand v0.0.0-20230731133715-1bf7a214788a
```

Comment:
Does the `replace` on line 5 make this unnecessary?

Reply:
This is a good question. It doesn't make sense to me to have any pinning to a specific hash/version when the dependency is filesystem based. I [removed it](https://github.com/algorand/go-algorand/pull/5621/commits/d157c2e52f45903e54c10401bd05490e393c0151) and `go mod tidy` + tests still work.